src/sites/dMCDA.jl

```diff
@@ -405,10 +407,10 @@ function guided_site_selection(
         rankingsin = [site_ids zeros(Int64, length(site_ids)) zeros(Int64, length(site_ids))]
     end
 
-    nsites::Int64 = length(site_ids)
+    n_sites::Int64 = length(site_ids)
 
     # if no sites are available, abort
-    if nsites == 0
+    if n_sites == 0
```

Comment:
This isn't really a comment on your PR, but should we have a warning message when this happens?

Reply:
I would say no in this case because sites can be legitimately filtered out, particularly under high DHW conditions?

Reply:
Thats true, so then we are using the previous time step's selection of sites

Reply:
Ah okay. We need to discuss under what circumstances deployments should be stopped then